### PR TITLE
[ISSUE #5092] add unit test for SystemConfig/TimeoutUtils/UrlAnalysis…

### PR DESCRIPTION
**Pull request by MajorHe1**
…Utils/ZipUtils in util.

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

for #5092 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    
+    @Test
+    public void testGetCurrentTimeStr() {
+        try {
+            Timestamp timestamp = TimeUtils.getCurrentTime();
+            Date date1 = new Date(timestamp.getTime());
+            System.out.println(date1.toString());
+            Date date2 = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(TimeUtils.getCurrentTimeStr());
+            System.out.println(date2.toString());
+        } catch (Exception e) {
```
Comment (li-xiao-shuang) [inline config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java:36]: 异常要抛出来

File: config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import java.util.Date;
+
+public class TimeUtilsTest {
+    
+    @Test
+    public void testGetCurrentTimeStr() {
+        try {
+            Timestamp timestamp = TimeUtils.getCurrentTime();
+            Date date1 = new Date(timestamp.getTime());
+            System.out.println(date1.toString());
```
Comment (li-xiao-shuang) [inline config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java:33]: 单元测试需要验证结果 不然没有意义

File: config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java (hunk 1 above)
Comment (MajorHe1) [reply config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java]: already updated

File: config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java (hunk 2 above)
Comment (MajorHe1) [reply config/src/test/java/com/alibaba/nacos/config/server/utils/TimeUtilsTest.java]: already updated

**Review by li-xiao-shuang — APPROVED** [review]

**Pull request merged**